experiment/measurer.py

```diff
@@ -69,10 +70,11 @@ def measure_main(experiment_config):
     max_total_time = experiment_config['max_total_time']
     measure_loop(experiment, max_total_time)
 
+    # Clean up resources.
```

Comment:
How sure are we this is needed?

Reply:
I was seeing some ooms before that don't happen anymore. We won't need this after measurer is broken into individual tasks.